docs/source/api/v1/letsencrypt_autorenew.rst

```diff
@@ -36,54 +36,20 @@ No parameters available
 
 Response Structure
 ------------------
-:LetsEncryptExpirations: A list of objects with information regarding certificate expiration for all Let's Encrypt certificates
-
-	:XmlId:       The :term:`Delivery Service`'s uniquely identifying :ref:`ds-xmlid`
-	:Version:     An integer that defines the "version" of the key - which may be thought of as the sequential generation; that is, the higher the number the more recent the key
-	:Expiration:  The expiration date of the certificate for the :term:`Delivery Service` in :rfc:`3339` format
-	:AuthType:    The authority type of the certificate for the :term:`Delivery Service`
-	:Error:       Any errors received in the renewal process
-
-:SelfSignedExpirations:  A list of objects with information regarding certificate expiration for all self signed certificates
-
-	:XmlId:       The :term:`Delivery Service`'s uniquely identifying :ref:`ds-xmlid`
-	:Version:     An integer that defines the "version" of the key - which may be thought of as the sequential generation; that is, the higher the number the more recent the key
-	:Expiration:  The expiration date of the certificate for the :term:`Delivery Service` in :rfc:`3339` format
-	:AuthType:    The authority type of the certificate for the :term:`Delivery Service`
-	:Error:       Any errors received in the renewal process
-
-:OtherExpirations:       A list of objects with information regarding certificate expiration for all other certificates
-
-	:XmlId:       The :term:`Delivery Service`'s uniquely identifying :ref:`ds-xmlid`
-	:Version:     An integer that defines the "version" of the key - which may be thought of as the sequential generation; that is, the higher the number the more recent the key
-	:Expiration:  The expiration date of the certificate for the :term:`Delivery Service` in :rfc:`3339` format
-	:AuthType:    The authority type of the certificate for the :term:`Delivery Service`
-	:Error:       Any errors received in the renewal process
 
 .. code-block:: http
 	:caption: Response Example
 
 	HTTP/1.1 200 OK
 	Content-Type: application/json
 
-	{ "response": {
-		"LetsEncryptExpirations": [
-			{
-				"XmlId":"demo2",
-				"Version":1,
-				"Expiration":"2020-08-18T13:53:06Z",
-				"AuthType":"Lets Encrypt",
-				"Error":null
-			}
-		],
-		"SelfSignedExpirations": [
-			{
-				"XmlId":"demo1",
-				"Version":3,
-				"Expiration":"2020-08-18T13:53:06Z",
-				"AuthType":"Self Signed",
-				"Error":null
-			}
-		],
-		"OtherExpirations":null
-	}}
+	{ "alerts": [
+		{
+			"text": "This endpoint is deprecated, please use letsencrypt/autorenew instead",
+			"level": "warning"
+		},
+		{
+			"text": "Beginning async call to renew certificates.  This may take a few minutes.",
+			"level": "success"
+		}
+	]}
```

Comment:
You can't change the response structure of old API versions. If there's really no way to meaningfully reproduce it, then you can just use meaningless data, but this is a breaking client change to a released API version.

Reply:
i think these were just bad docs. the other way was the original plan but then it took too long so i switched it to be async and the other info was sent in an email instead of as the response. i dont think that version was ever released

Reply:
alright, in that case: good work fixing it